diff --git a/node_modules/@novnc/novnc/lib/util/browser.js b/node_modules/@novnc/novnc/lib/util/browser.js
index 8121bc1..c88392d 100644
--- a/node_modules/@novnc/novnc/lib/util/browser.js
+++ b/node_modules/@novnc/novnc/lib/util/browser.js
@@ -176,7 +176,9 @@ function _checkWebCodecsH264DecodeSupport2() {
   }));
   return _checkWebCodecsH264DecodeSupport2.apply(this, arguments);
 }
-exports.supportsWebCodecsH264Decode = supportsWebCodecsH264Decode = await _checkWebCodecsH264DecodeSupport();
+_checkWebCodecsH264DecodeSupport().then(function (result) {
+  exports.supportsWebCodecsH264Decode = supportsWebCodecsH264Decode = result;
+});
 
 /*
  * The functions for detection of platforms and browsers below are exported
